packages/blocks/drawing-canvas/src/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { TDDocument, TDExport, TldrawApp } from "@tldraw/tldraw";
+
+/**
+ * @todo implement endpoint for handling image exports
+ * This only handles svg exports at the moment
+ * @see https://github.com/tldraw/tldraw/blob/main/apps/www/utils/export.ts
+ * @see https://github.com/tldraw/tldraw/blob/main/apps/www/pages/api/export.ts
+ */
+
+export async function handleExport(info: TDExport) {
+  if (info.serialized) {
+    const link = document.createElement("a");
+    link.href = `data:text/plain;charset=utf-8,${encodeURIComponent(
+      info.serialized,
+    )}`;
+    link.download = `${info.name}.${info.type}`;
+    link.click();
+  }
+}
+
+export const getInitialDocument = (document: string | undefined, entityId) => {
```

Comment:
Minor: let's explicitly define return type of this function (and the one above).

Reply:
Done here => https://github.com/hashintel/hash/pull/515/commits/229dfe48559f79788dac315167cd7ad95af46289.
The one above doesn't return anything

Reply:
If a function does not return anything, it is still useful to define its return type (`: void`). This is not a strict rule, but it helps with linting and speeds up TS a bit.

Reply:
I have no strong feelings about it. Updated here => [ad7d9b3](https://github.com/hashintel/hash/pull/515/commits/ad7d9b337fa1b24166449b08b311dd4373b28c13)